include/LLGL/Container/Set.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Set.h
+ *
+ * Copyright (c) 2015 Lukas Hermanns. All rights reserved.
+ * Licensed under the terms of the BSD 3-Clause license (see LICENSE.txt).
+ */
+
+#ifndef LLGL_SET_H
+#define LLGL_SET_H
+
+#include <LLGL/Container/STLAllocator.h>
+
+#include <set>
+#include <utility>
+
+namespace LLGL{
```

Comment:
We can look into getting a `.clang-format` up an running as you suggested. I am personally not a huge fan of auto-formatters, but I can see why it makes sense for public projects like this one. Until then, would you mind changing the code indentation to [Allman](https://en.wikipedia.org/wiki/Indentation_style#Allman) style?